etc/config/c++.amazon.properties

```diff
@@ -223,59 +223,63 @@ group.clang.compilerCategories=clang
 compiler.clang30.exe=/opt/compiler-explorer/clang+llvm-3.0-x86_64-linux-Ubuntu-11_10/bin/clang++
 compiler.clang30.alias=/usr/bin/clang++
 compiler.clang30.semver=3.0.0
+compiler.clang30.options=-I/opt/compiler-explorer/gcc-4.6.4/lib/gcc/x86_64-linux-gnu/4.6.4/include -I/opt/compiler-explorer/gcc-4.6.4/include/c++/4.6.4 -I/opt/compiler-explorer/gcc-4.6.4/include/c++/4.6.4/x86_64-linux-gnu/
 compiler.clang30.options=
```

Comment:
Can you remove this line:


```
################
## c++.amazon.properties
duplicate_lines:
  Line 227: compiler.clang30.options
```

(you can test with `python3 ./etc/scripts/util/propscheck.py` or let the pre-commit hooks run)

Reply:
Fixed

Reply:
Thank you!! :) will merge once green. plan on doing a site update later today.